pkg/build/controller/strategy/sti.go

```diff
@@ -146,6 +156,10 @@ func (bs *SourceBuildStrategy) CreateBuildPod(build *buildv1.Build, includeAddit
 					Name:      "buildworkdir",
 					MountPath: buildutil.BuildWorkDirMount,
 				},
+				{
+					Name:      "buildcachedir",
+					MountPath: "/var/lib/containers/cache/",
+				},
```

Comment:
this one *is* needed (we will pull images from within this container). And docker.go should have it too.

assuming you care about this cache for pulling images (not pushing them).